docs/user/dashboard/dashboard.asciidoc

```diff
@@ -262,22 +309,49 @@ Copy panels from one dashboard to another dashboard.
 . On the *Copy to dashboard* window, select the dashboard, then click *Copy and go to dashboard*.
 
 [float]
-[[explore-the-underlying-documents]]
-== Explore the underlying documents
+[[search-or-filter-your-data]]
+== Search and filter your data
 
-You can add additional interactions that allow you to open *Discover* from dashboard panels. To use the interactions, the panel must use only one index pattern.
+{kib} supports several ways to search your data and apply {es} filters. You can combine the filters with any panel
+filter to display the data want to you see.
 
-Panel interaction:: Opens the data in *Discover* with the current dashboard filters, but does not take the filters
-saved with the panel.
-+
-To enable panel interactions, refer to <<settings-explore-data-in-context,`xpack.discoverEnhanced.actions.exploreDataInContextMenu.enabled`>>.
-+
-NOTE: In {kib} 7.13 and earlier, the panel interaction is enabled by default.
+For more information about {kib} and {es} filters, refer to <<kibana-concepts-analysts,{kib} concepts>>.
 
-Series interaction::
-Opens the series data in *Discover* from inside the panel.
-+
-To enable series interactions, refer to <<settings-explore-data-in-chart,`xpack.discoverEnhanced.actions.exploreDataInChart.enabled`>>.
+To apply a panel-level time filter:
+
+. Open the panel menu, then select *More > Customize time range*.
+
+. Enter the time range you want to view, then click *Add to panel*.
+
+[float]
+[[apply-design-options]]
+== Apply design options
+
+Apply a set of design options to the entire dashboard.
+
+. If you're in view mode, click *Edit* in the toolbar.
+
+. In the toolbar, *Options*, then use the following options:
+
+* *Use margins between panels* &mdash; Adds a margin of space between each panel.
+
+* *Show panel titles* &mdash; Displays the titles in the panels headers.
+
+* *Sync color pallettes across panels* &mdash; Applies the same color palette to all panels on the dashboard.
+
+[float]
+[[save-dashboards]]
+== Save dashboards
```

Comment:
Not sure how to document this, but there is a quicksave feature now. So:
If the dashboard is new / unsaved, the button will read `save`
If the dashboard is saved and being edited, the save button will read `save as` and there will be another `save` button which saves the dashboard without asking for a title.